ern-local-cli/src/commands/create-api-impl.js

```diff
@@ -66,6 +66,12 @@ exports.handler = async function ({
   hasConfig: boolean,
   skipNpmCheck?: boolean
 }) {
+  // Fixes https://github.com/electrode-io/electrode-native/issues/265
+  if (!await cliUtils.doesPackageExistInNpm(api)) {
```

Comment:
Can you move this block in the `try` block down below, and throw an error instead of logging an error and returning ?

```javascript
throw new Error(`Couldn't find package ${api} to generate the api implementation`) 
```

This is the direction we'll favor and that I started taking in a recent PR. All commands when failing should log the error and exit the process with a non zero status code (to make it easier for running automated system tests is one of the reason, but also to keep things coherent and conventional). The catch block in the command will call the function that just does that.